lib/cache/redis.ex

```diff
@@ -186,11 +218,48 @@ defmodule Cache.Redis do
   end
 
   def hash_delete(pool_name, key, field, opts) do
+    field = TermEncoder.encode(field, opts[:compression_level])
     command(pool_name, ["HDEL", cache_key(pool_name, key), field], opts)
   end
 
   def hash_values(pool_name, key, opts) do
-    command(pool_name, ["HVALS", cache_key(pool_name, key)], opts)
+    with {:ok, data} <- command(pool_name, ["HVALS", cache_key(pool_name, key)], opts) do
+      values =
+        Enum.map(data, fn value ->
+          TermEncoder.decode(value)
+        end)
+
+      {:ok, values}
+    end
+  end
+
+  def find(pool_name, find_opts, _opts) do
+    match = Keyword.get(find_opts, :match, "*")
```

Comment:
instead you should use `find_opts[:match] || "*"` the reason why is because if the value is set to nil, the default isn't used and you'll get a nil value